cypress/e2e/login.cy.tsx

```diff
@@ -0,0 +1,17 @@
+describe("Login", () => {
+  beforeEach(() => {
+    cy.visit("/login");
+  });
+
+  it("renders login form", () => {
```

Comment:
put login and signup in auth.cy.tsx

test thing by scenario, not by action unit

test what happens when user click on submit after typing BAD format value (test zod validation)